lib/tasks/queue.rake

```diff
@@ -1,3 +1,4 @@
+class ValueError < RuntimeError; end
```

Comment:
Where was this defined before, when it was called from line 34?

Reply:
It wasn't defined anywhere.

Reply:
This is probably my fault as I wrote the original. I must have been thinking of ArgumentError and didn't test this case. I don't think there's any need to introduce a new class, just raise something sensible instead. 🙂

Reply:
@MatMoore have changed